site/index.html

```diff
@@ -76,6 +76,51 @@ <h2>Try it!</h2>
 
 <br>
 
+<div class="features">
+   <div class=" container ">
+      <h2>Why Haskell?</h2>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Maintain with confidence</h3>
+            <p>Fearless progress and refactoring over the long term. Haskell has one of the strongest type systems on the market to prevent mistakes
+               before they reach production, saving on total software lifetime costs. “If it compiles, it works.”&mdash;a Haskell proverb.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Scale with ease</h3>
+            <p>From 1 to 100k users, and more. Haskell can take advantage of multi-core processors with ease and confidence
+               like no mainstream language can, and offers one of the best support for async, concurrent and parallel programming.
+            </p>
+         </div>
+      </div>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Rapid prototyping</h3>
+            <p>Start small and explore new ideas to find your market fit. Haskell is a compact language with a large ecosystem of open-source libraries and tools
+               to get you started and keep you growing.
+               The strong type system makes it easy to write small and large programs that work, even when you don't like writing types &mdash; the compiler will figure it out.
+            </p>
+         </div>
```

Comment:
These two are very good hooks, I'd put them higher in the list. Maintainability is nice but it's not a good way to attract people (although it is a very good way to keep them).

Reply:
I'm open to reorder the points later on as my personal preference is like your's. However, SPJ seems to favor maintainability first: https://discourse.haskell.org/t/emphasize-why-haskell-on-haskell-org-landing-page/12036/12?u=malteneuss Let's wait for a consensus.

Reply:
SPJ also says "I’m an academic, so my opinion on Haskell-in-the-wild is deeply suspect".

As an industry professional who has studied other programming languages' marketing, I can assure you the "hook" of a PL page is necessary to attract people by expressing things like motive force, building, and deploying. Maintainability doesn't speak to anyone until they have a working code base that they deploy or distribute. 

To make a video game analogy, it would be wasteful to mention endgame strategy to a new player who has yet to create their character. There is a fundamental mismatch in terms of where one is in the timeline.

Reply:
Agreed.